cves/kernel/CVE-2020-2732.yml

```diff
@@ -55,7 +55,13 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
```

Comment:
I think you should explain what L0, L1, and L2 mean. The audience might not understand what that means.

Reply:
Cleared up the logic! Upon further inspection what I wrote was completely wrong.